scheduleit/views/meetings/show.twig

```diff
@@ -120,20 +120,8 @@
             N/A
           {% endif %}
         </td>
-        <td style="display:none;">
-          {{site_url}}/meetings/remove_attendee
-        </td>
-        <td style="display:none;">
-          {{meeting.attendee_onid}}
-        </td>
-        <td style="display:none;">
-          {{meeting.timeslot_hash}}
-        </td>
-        <td style="display:none;">
-          {{meeting.name}}
-        </td>
         <td class="col-1">
-          <button class="table-btn">
+          <button class="table-btn" data-name="{{meeting.name}}" data-timeslot-hash="{{meeting.timeslot_hash}}" data-attendee-onid="{{meeting.attendee_onid}}" data-url="{{site_url}}/meetings/remove_attendee">
```

Comment:
If we want to have to have this button look a little cleaner, we could do:
```
<i
  class="table-btn fa fa-times text-danger"
  data-name="{{meeting.name}}"
  data-timeslot-hash="{{meeting.timeslot_hash}}"
  data-attendee-onid="{{meeting.attendee_onid}}"
  data-url="{{site_url}}/meetings/remove_attendee"
  role="button"
</i>
```
![Screen Shot 2020-07-31 at 4 25 12 PM](https://user-images.githubusercontent.com/15021/89087824-78720380-d34a-11ea-841a-03f32d7838e0.png)

OR
```
<button
  class="table-btn btn btn-danger btn-sm"
  data-name="{{meeting.name}}"
  data-timeslot-hash="{{meeting.timeslot_hash}}"
  data-attendee-onid="{{meeting.attendee_onid}}"
  data-url="{{site_url}}/meetings/remove_attendee"
  type="button"
>
  <i class="fa fa-times"></i>
</button> 
```
![Screen Shot 2020-07-31 at 4 24 57 PM](https://user-images.githubusercontent.com/15021/89087829-7c9e2100-d34a-11ea-9caa-b6e9d0fbfc2b.png)